docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  db:
+    image: postgres
```

Comment:
Please. add the image tag as well. The tag specifies the PostgreSQL version and this way we can make sure everyone is working on the same version. In this case the image (with the tag) should be `postgres:15-alpine`